### PR TITLE
tests, network: Replace run migration function

### PR DESCRIPTION
**Pull request by RamLavi**
**What this PR does / why we need it**:
`runMigrationAndExpectCompletion` function is implemented both locally and on migration package.
In order to align with other migration tests, replacing function with equivalent on migration package.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: tests/network/vmi_networking.go [hunk 1]
```diff
@@ -879,9 +859,9 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 				podIP := libnet.GetPodIPByFamily(virtHandlerPod, k8sv1.IPv4Protocol)
 				Expect(ping(podIP)).To(Succeed())
 
-				By("Execute migration")
+				By("starting the migration")
```
Comment (orelmisan) [inline tests/network/vmi_networking.go:862]: nit:
```suggestion
				By("Starting the migration")
```

File: tests/network/vmi_networking.go [hunk 2]
```diff
@@ -921,9 +901,9 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 				podIP := libnet.GetPodIPByFamily(virtHandlerPod, k8sv1.IPv6Protocol)
 				Expect(ping(podIP)).To(Succeed())
 
-				By("Execute migration")
+				By("starting the migration")
```
Comment (orelmisan) [inline tests/network/vmi_networking.go:904]: nit:
```suggestion
				By("Starting the migration")
```

**Review by orelmisan — APPROVED** [review]

**Review by xpivarc — COMMENTED** [review]
/approve

File: tests/network/vmi_networking.go (hunk 1 above)
Comment (RamLavi) [reply tests/network/vmi_networking.go:862]: it's used both with/without capital all over [migration_test.go ](https://github.com/kubevirt/kubevirt/blob/cd3239c5185f8b1e5dfcf5feeb14b853b347e7de/tests/migration_test.go). Let's fix it together on another PR

File: tests/network/vmi_networking.go (hunk 2 above)
Comment (RamLavi) [reply tests/network/vmi_networking.go:904]: same

**Review by EdDev — APPROVED** [review]
I do not really like the dependency on the overloaded `tests` package, but this could/should be changed in a different change.

**Pull request merged**